phoenix/core/models.py

```diff
@@ -148,9 +148,25 @@ def created_timestamp(self):
 
 
 class Outage(AbstractOutage):
+    # If no ETA, phoenix will ask assignee for ETA update after X minutes.
+    # This attribute should only be set to True if new outage has not been provided
+    # with ETA. In case of edit or some other change of ETA this should be False.
+    #
+    # User should be prompted for ETA update only if ETA wasn't known at the time of
+    # announcement creation.
+    prompt_for_eta_update = models.BooleanField(default=True)
+    # If false users won't be able to change ETA using the prompt menu
+    prompt_active = models.BooleanField(default=True)
+
     def __str__(self):
         return f"Outage {self.id}"
 
+    def enable_eta_update_prompt(self):
+        self.prompt_for_eta_update = True
+
+    def disable_eta_update_prompt(self):
+        self.prompt_for_eta_update = False
```

Comment:
These methods add very little value and they are an extra layer of abstraction which I'd prefer to avoid.